<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавить новое слово</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Язык:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="language_combo"/>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Слово:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="original_edit"/>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Перевод:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="translation_edit"/>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Пример использования (опционально):</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="example_edit"/>
   </item>
<item>
 <widget class="QLabel" name="label_5">
  <property name="text">
   <string>Произношение (MP3):</string>
  </property>
 </widget>
</item>
<item>
 <layout class="QHBoxLayout" name="horizontalLayout">
  <item>
   <widget class="QLineEdit" name="pronunciation_edit">
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </item>
  <item>
   <widget class="QPushButton" name="browse_btn">
    <property name="text">
     <string>Обзор...</string>
    </property>
   </widget>
  </item>
 </layout>
</item>
   <item>
    <widget class="QDialogButtonBox" name="button_box">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>